server/config/config.go

```diff
@@ -250,3 +257,24 @@ func MakeConfigParser() (*Parser, error) {
 	fs.IntVar(&cfg.HTTPPort, "http-port", defaultHTTPPort, "port of http server")
 	return builder, nil
 }
+
+// MakeConfigParseFromToml read configuration from the toml file, if the config item already exists, it will be overwritten.
+func MakeConfigParseFromToml(conf *Config) error {
```

Comment:
We need a better name:
```suggestion
func ParseConfigFromFile(conf *Config) error {
```

BTW, will this cause start failure if the command line doesn't contain the config file path?

Reply:
> We need a better name:
> 
> BTW, will this cause start failure if the command line doesn't contain the config file path?

Server will skip config parse if config file path not specified, it will be started in the way that there is no configuration file before.